main.py

```diff
@@ -78,6 +78,8 @@
     parser.add_argument('--dmtet_reso_scale', type=float, default=8, help="multiply --h/w by this for dmtet finetuning")
     # controlnet
     parser.add_argument('--guidance_image_path', type=str, default="")
+    parser.add_argument('--guidance_image_view', type=float, default=0, help="Refer nerf/provider.py for what values represent. Default is 0 (front).")
```

Comment:
should the type actually be `int` instead of `float`?

Reply:
yes that's right! Good catch